src/System.CommandLine.Tests/CompletionContextTests.cs

```diff
@@ -33,6 +36,128 @@ public void CommandLineText_preserves_command_line_prior_to_splitting_when_compl
                        .Be(commandLine);
         }
 
+        [Fact]
+        public void CommandLineText_is_parsed_when_option_other_than_last_is_in_name_equals_sign_value_format()
```

Comment:
This test name is a little hard to understand, and `CommandLinetext_is_parsed` isn't very specific about the expectation. I think the expectation is that the tokens don't include the delimiters? (Also, that it doesn't throw, but that typically goes without saying.) 

A more sentence-style test name is encouraged, e.g. something like `When_equals_sign_is_used_as_delimiter_then_it_is_not_included_in_tokens`.
 
Some of these tests seem to differ only by the arguments being parsed, so combining them into a `[Theory]` might give you fewer tests to name.